spec.html

```diff
@@ -7598,7 +7598,7 @@
         <emu-alg>
           1. Let _args_ be the arguments object.
           1. If _P_ is `"caller"`, then
-            1. Let _desc_ be ! OrdinaryGetOwnProperty(_args_, _P_).
+            1. Let _desc_ be OrdinaryGetOwnProperty(_args_, _P_).
             1. If IsDataDescriptor(_desc_) is *true*, return *true*.
```

Comment:
Oh I have that wrong. Only if a completion record is returned it will unwrap, otherwise it'll leave the return value be. Ugh.